lessons/intro/pyqt/index.md

```diff
@@ -476,8 +476,8 @@ Nyní by již mapa měla být v okně vidět barevně.
 
 Protože barvičky jsou příliš nudné, přidáme do mapového widgetu obrázky.
 
-Veškerou ke cvičení i k úkolu potřebnou grafiku najdete na [GitHubu](https://github.com/pyvec/naucse.python.cz/tree/master/lessons/intro/pyqt/static/pics).
-Je k dispozici pod public domain (tj. „dělej si s tím, co chceš“), pochází ze studia [Kenney],
+Veškerou, ke cvičení i k úkolu potřebnou, grafiku najdete na [GitHubu](https://github.com/pyvec/naucse.python.cz/tree/master/lessons/intro/pyqt/static/pics).
```

Comment:
Záleží, ale myslím, že s čárkama se to líp čte a nezbytně nutná informace (aby to muselo být bez čárek) to není.